vip-scanner/config-vip-scanner.php

```diff
@@ -54,6 +55,7 @@
 ) );
 
 VIP_Scanner::get_instance()->register_review( 'VIP Theme Review', array(
+	'BatCacheVariantCheck',
```

Comment:
There seems to be a typo in these class names - the second `C` is capitalized, but not in the class names (shouldn't be capitalized anywhere).